x-pack/plugins/maps/public/classes/layers/tiled_vector_layer/tiled_vector_layer.tsx

```diff
@@ -115,7 +115,7 @@ export class TiledVectorLayer extends VectorLayer {
     dataFilters,
   }: DataRequestContext) {
     const requestToken: symbol = Symbol(`layer-${this.getId()}-${SOURCE_DATA_REQUEST_ID}`);
-    const searchFilters: VectorSourceRequestMeta = await this._getSearchFilters(
+    const searchFilters: VectorSourceRequestMeta = await this._getVectorSourceRequestMeta(
```

Comment:
How about renaming variable to requestMeta to better match function name change?